extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -215,4 +215,11 @@ private LDAPGuacamoleProperties() {}
         public String getName() { return "ldap-user-attributes"; }
 
     };
+    
+    public static final StringGuacamoleProperty LDAP_MEMBER_ATTRIBUTE = new StringGuacamoleProperty() {
```

Comment:
Please document.

Reply:
Wow, not sure why I forgot that.  Documented.